feature-libs/asm/assets/translations/en/asm.json

```diff
@@ -147,7 +148,6 @@
       "success": "Cart has been successfully assigned",
       "assignCartId": "Assign a cart id to customer",
       "enterCartId": "Enter cart id",
-      "resetCartId": "Reset",
```

Comment:
don't remove the existing translation, in case customer are using it.

Reply:
have been modified.